thorchain/assetlist.json

```diff
@@ -232,8 +232,43 @@
       }
     },
     {
-      "base": "x/nami-index-fixed-thor1yqf5spdv8c4088zmvqsg32eq63fzepsjvntahdk0ek0yjnkt3qdqftp3lc-rcpt",
+      "description": "sTCY is the auto-compounding version of the TCY token - THORChain Yield - to address the default of THORFi. 10% of all system income is paid to TCY stakers.",
+      "denom_units": [
+        {
+          "denom": "stcy",
+          "exponent": 0
+        },
+        {
+          "denom": "x/staking-tcy",
+          "exponent": 0
+        }
+      ],
       "type_asset": "sdk.coin",
+      "base": "x/staking-tcy",
+      "name": "THORChain sTCY",
+      "display": "stcy",
+      "symbol": "sTCY",
```

Comment:
Please either define the relationship to TCY (using `traces`), or else use all UPPERCASE letters in the symbol. 
e.g., https://github.com/cosmos/chain-registry/blob/master/thorchain/assetlist.json#L284-L291